dsl_parser/import_resolver/abstract_import_resolver.py

```diff
@@ -39,14 +39,8 @@ class AbstractImportResolver(object):
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
-    def resolve(self, import_url):
-        raise NotImplementedError
-
     def fetch_import(self, import_url):
-        url_parts = import_url.split(':')
-        if url_parts[0] in ['http', 'https', 'ftp', 'file']:
-            return self.resolve(import_url)
-        return read_import(import_url)
+        raise NotImplementedError
```

Comment:
i think we used to, or still do, provide a way for the user to pass their own import resolvers, right? if so, the abstract import resolver is public api, and we should keep some backwards compat here: if the user used to implement just `resolve`, because that was all that was needed, this change will break their resolver... can we do it in a compatible way?